### PR TITLE
/services and /services/namespace/name

### PR DESCRIPTION
**Pull request by jasmingacic**
added /services and /services/namespace/name implementation 
plus some additional changes to the OpenAPI spec

Signed-off-by: Jasmin Gacic <jasmin.gacic@gmail.com>

@topliceanurazvan you can rebase #4 to my branch and then add needed changes

### CODE REVIEW

**Comment by jasmingacic** [comment]
I extended `/apis` with fleet params as requested here https://github.com/kubeshop/kuskgateway-api-server/pull/5#issuecomment-1070972033

File: server/go/api_services_service_impl.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+		opts, err := spec.GetOptions(apiSpec)
+		if err != nil {
+			return Response(http.StatusInternalServerError, nil), err
+		}
+
+		_, err = s.kuskClient.GetSvc(opts.Upstream.Service.Namespace, opts.Upstream.Service.Name)
+		status := "available"
+		if err != nil {
+			fmt.Println(err)
```
Comment (kylehodgetts) [inline server/go/api_services_service_impl.go:59]: Should we use a log package for logging out errors ?

File: server/kubeconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```
Comment (kylehodgetts) [inline server/kubeconfig:1]: does this need to be in git?

**Review by kylehodgetts — APPROVED** [review]
Some small comments but looks good on the whole

File: server/kubeconfig (hunk 2 above)
Comment (jasmingacic) [reply server/kubeconfig:1]: It is a private repo for testing. I will have it removed once we go public

Comment (jasmingacic) [reply server/kubeconfig:1]: Ah it is public :D

Comment (jasmingacic) [reply server/kubeconfig]: removed ti

**Pull request merged**